### DISCUSSION TITLE
What is the difference between these browser branches?

### DISCUSSION THREAD

**Discussion by Someone-G** [CookieJarApps/SmartCookieWeb-Preview#136] [Q&A]
Hello. What is the difference between these browser branches?

1. https://github.com/CookieJarApps/SmartCookieWeb
2. https://github.com/CookieJarApps/SmartCookieWeb-Preview

**Comment by CookieJarApps** [comment]
SmartCookieWeb is based on WebView, a component built into Android that automatically updates through Google Play and uses Chrome's rendering engine to render web pages. It's got the most features, but does not have extension support.

SmartCookieWeb Preview is based on GeckoView (which is built into the app, rather than being external) and uses the same web rendering engine as Firefox. It's less stable and has fewer features, but has support for Firefox extensions and has more active development.

**Comment by Someone-G** [comment]
Has the development of the SmartCookieWeb browser been stopped completely?
